docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,589 @@
+
+> While the Full Route Cache only caches statically rendered routes, the Router Cache applies to both statically and dynamically rendered routes.
+
+> **Related Terms:**
+>
+> You may see the Router Cache being referred to as **Client-side Cache** or **Prefetch Cache**. While **Prefetch Cache** refers to the prefetched route segments, **Client-side Cache** refers to the whole Router cache, which includes both visited and prefetched segments.
+> This cache specifically applies to Next.js and Server Components, and is different to the browser's [bfcache](https://web.dev/bfcache/), though it has a similar result.
+
+### Duration
+
+The cache is stored in the browser's temporary memory. Two factors determine how long the router cache lasts:
+
+- **Session**: The cache persists across navigation. However, it's cleared on page refresh.
+- **Automatic Invalidation Period**: The cache of an individual segment is automatically invalidated after a specific time. The duration depends on whether the route is [statically](/docs/app/building-your-application/rendering/static-and-dynamic#static-rendering-default) or [dynamically](/docs/app/building-your-application/rendering/static-and-dynamic#dynamic-rendering) rendered:
+  - **Dynamically Rendered**: 30 seconds
+  - **Statically Rendered**: 5 minutes
+
+While a page refresh will clear **all** cached segments, the automatic invalidation period only affects the individual segment from the time it was last accessed or created.
+
+By adding `prefetch={true}` or calling `router.prefetch` for a dynamically rendered route, you can opt into caching for 5 minutes.
+
+### Invalidation
+
+There are two ways you can invalidate the Router Cache:
+
+- In a **Server Action**:
+  - Revalidating data on-demand by path with ([`revalidatePath`](/docs/app/api-reference/functions/revalidatePath)) or by cache tag with ([`revalidateTag`](/docs/app/api-reference/functions/revalidateTag))
+  - Using [`cookies.set`](/docs/app/api-reference/functions/cookies#cookiessetname-value-options) or [`cookies.delete`](/docs/app/api-reference/functions/cookies#deleting-cookies) invalidates the Router Cache to prevent routes that use cookies from becoming stale (e.g. authentication).
+- Calling [`router.refresh`](/docs/app/api-reference/functions/use-router) will invalidate the Router Cache and make a new request to the server for the current route.
+
+### Opting out
+
+It's not possible to opt out of the Router Cache.
+
+You can opt out of **prefetching** by setting the `prefetch` prop of the `<Link>` component to `false`. However, this will still temporarily store the route segments for 30s to allow instant navigation between nested segments, such as tab bars, or back and forward navigation. Visited routes will still be cached.
```

Comment:
praise: thanks for documenting this behaviour 🙏 - it was a point of confusion when trying things out up until now.

thought: since we can't opt-out of the Router Cache, 30s seems to be the absolute minimum that a page will be cached in the Router Cache. While I agree that this is a good default, the fact that there is no way to change this is a bit concerning. I talked to Tim a while back and he mentioned that very likely, defining a `revalidate` time on a route segment that is _below_ 30s would also lower the caching time for the Router Cache. Was this idea discarded ?